File: benchmarks/python/sort_numbers.py
import sys

def main():
    n = int(sys.argv[1]) if len(sys.argv) > 1 else 300000
    numbers = []
    seed = 1
    for i in range(n):
        seed = (seed * 75 + 74) % 65537
        numbers.append(seed)
    numbers.sort()
    print(numbers[0], numbers[n // 2], numbers[n - 1])

main()

File: benchmarks/ruby/sort_numbers.rb
n = (ARGV[0] || "300000").to_i
numbers = []
seed = 1
n.times do
  seed = (seed * 75 + 74) % 65537
  numbers.push(seed)
end
numbers.sort!
puts "#{numbers[0]} #{numbers[n / 2]} #{numbers[n - 1]}"
